app/src/providers/notificationTrackingProvider.tsx

```diff
@@ -0,0 +1,41 @@
+import messaging from '@react-native-firebase/messaging';
```

Comment:
address [SEL-344](https://linear.app/selfprotocol/issue/SEL-344/implement-onnotificationopenedapp-for-react-native-firebase)